pkg/api/annotation_key_constants.go

```diff
@@ -29,6 +29,9 @@ const (
 	// MirrorAnnotationKey represents the annotation key set by kubelets when creating mirror pods
 	MirrorPodAnnotationKey string = "kubernetes.io/config.mirror"
 
+	// HostNameAnnotationKey represents the annotation key of hostname set by kubelets
+	HostNameAnnotationKey string = "kubernetes.io/hostname"
```

Comment:
yes, it's label. I'm a little confused on where put such kind of const var, started a tech-debt at #48038.